config/zones.json

```
@@ -3069,7 +3069,7 @@
       "https://github.com/alixunderplatz"
     ],
     "parsers": {
-      "production": "KW.fetch_production"
+      "production": "KW.fetch_consumption"
```

This should also change to consumption! sorry I missed this the first time!
```suggestion
      "consumption": "KW.fetch_consumption"
```